core/src/main/resources/lib/hudson/executors.jelly

```diff
@@ -39,8 +39,9 @@ THE SOFTWARE.
       <j:if test="${!c.acceptingTasks}"> <st:nbsp/> (${%suspended})</j:if>
     </d:tag>
     <d:tag name="executor">
+      <j:set var="ae" value="${e.asynchronousExecution}"/>
```

Comment:
@Peggs At the top of any page in this project, click the "Unwatch" button and choose "Not watching".